### ISSUE TITLE
[Cluster] Strange displayed size, when instance fetch store from master

### ISSUE THREAD

**Issue by FylmTM** [neo4j/neo4j#3585]
See last line:
```
2014-12-02 11:36:38.794+0000 INFO  [Cluster] Checking store consistency with master                                                                                                                                                          
2014-12-02 11:36:38.922+0000 INFO  [Cluster] The store is inconsistent. Will treat it as branched and fetch a new one from the master                                                                                                        
2014-12-02 11:36:40.196+0000 INFO  [Cluster] ServerId 2, moving to slave for master ha://10.133.234.120:6001?serverId=3                                                                                                                      
2014-12-02 11:36:40.294+0000 INFO  [Cluster] Copying store from master                                                                                                                                                                       
2014-12-02 11:36:40.496+0000 INFO  [Cluster] Copying index/lucene-store.db                                                                                                                                                                   
2014-12-02 11:36:40.498+0000 INFO  [Cluster] Copied  index/lucene-store.db 40.00 B                                                                                                                                                           
2014-12-02 11:36:40.499+0000 INFO  [Cluster] Copying neostore.propertystore.db.strings                                                                                                                                                       
2014-12-02 11:37:17.614+0000 INFO  [Cluster] Copied  neostore.propertystore.db.strings -1609969152.00 B                                                                                                                                      
```

Store in filesystem looks like this:
```
[root@neo4j-db-3 neo4j-database]# ls -lah | grep neostore
-rw-rw-r-- 1 neo4j neo4j   81 2014-12-02 06:42 neostore
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.id
-rw-rw-r-- 1 neo4j neo4j   15 2014-12-01 04:35 neostore.labeltokenstore.db
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.labeltokenstore.db.id
-rw-rw-r-- 1 neo4j neo4j  152 2014-12-01 04:35 neostore.labeltokenstore.db.names
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.labeltokenstore.db.names.id
-rw-rw-r-- 1 neo4j neo4j 129M 2014-12-01 04:53 neostore.nodestore.db
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.nodestore.db.id
-rw-rw-r-- 1 neo4j neo4j   68 2014-12-01 04:35 neostore.nodestore.db.labels
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.nodestore.db.labels.id
-rw-rw-r-- 1 neo4j neo4j 1,3G 2014-12-02 06:42 neostore.propertystore.db
-rw-rw-r-- 1 neo4j neo4j 416M 2014-12-01 04:35 neostore.propertystore.db.arrays
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.propertystore.db.arrays.id
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.propertystore.db.id
-rw-rw-r-- 1 neo4j neo4j  24K 2014-12-01 04:42 neostore.propertystore.db.index
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.propertystore.db.index.id
-rw-rw-r-- 1 neo4j neo4j 101K 2014-12-01 04:42 neostore.propertystore.db.index.keys
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.propertystore.db.index.keys.id
-rw-rw-r-- 1 neo4j neo4j 2,6G 2014-12-02 06:42 neostore.propertystore.db.strings
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.propertystore.db.strings.id
-rw-rw-r-- 1 neo4j neo4j 2,7M 2014-12-01 04:35 neostore.relationshipgroupstore.db
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.relationshipgroupstore.db.id
-rw-rw-r-- 1 neo4j neo4j 357M 2014-12-01 04:35 neostore.relationshipstore.db
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.relationshipstore.db.id
-rw-rw-r-- 1 neo4j neo4j   20 2014-12-01 04:35 neostore.relationshiptypestore.db
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.relationshiptypestore.db.id
-rw-rw-r-- 1 neo4j neo4j  190 2014-12-01 04:35 neostore.relationshiptypestore.db.names
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.relationshiptypestore.db.names.id
-rw-rw-r-- 1 neo4j neo4j  256 2014-12-01 04:35 neostore.schemastore.db
-rw-r--r-- 1 neo4j neo4j    9 2014-12-02 06:35 neostore.schemastore.db.id
```

**Comment by jakewins** [comment]
Hah, it's rolled over the max java integer size, I guess we use an int to represent that size in the diagnostics tool. Marking as a bug.

**Comment by spacecowboy** [comment]
This was fixed in 5466956
